Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -540,4 +548,41 @@ void ABenchmarkGymGameModeBase::SetLifetime(int32 Lifetime)
 	{
 		UE_LOG(LogBenchmarkGymGameModeBase, Warning, TEXT("Could not set NFR test liftime to %d. Timer was locked."), Lifetime);
 	}
-}
\ No newline at end of file
+}
+
+int ABenchmarkGymGameModeBase::GetAuthoritativePlayers()
+{
+	int PlayerCount = 0;
+	for (FConstPlayerControllerIterator Iterator = GetWorld()->GetPlayerControllerIterator(); Iterator; ++Iterator)
+	{
+		APlayerController* PlayerActor = Iterator->Get();
+		if (PlayerActor && PlayerActor->PlayerState && !MustSpectate(PlayerActor) && PlayerActor->HasAuthority())
```

Comment:
Could you add explicit nullptr checks: PlayerActor != nullptr  && PlayerActor->PlayerState != nullptr.

 It's just our style in the GDK

Reply:
If possible: const APlayerController* PlayerActor.

Reply:
Good style. 
We can't use const APlayerController* PlayerActor because MustSpectate is only supported APlayerController*.

Reply:
Ahh yeah, feels bad man.